pkg/storage/unified/resource/server.go

```diff
@@ -646,6 +661,27 @@ func (s *server) Create(ctx context.Context, req *resourcepb.CreateRequest) (*re
 	ctx, span := tracer.Start(ctx, "resource.server.Create")
 	defer span.End()
 
+	// check quotas and log for now
+	if s.quotaService != nil {
```

Comment:
For now, on create will validate the resource count against the quota, and log if they've exceeded it.